webportal/src/app/job/job-view/job-detail-table.component.ejs

```diff
@@ -79,7 +79,20 @@
           <td><%= taskRoles[taskRole].taskStatuses[i].containerId %></td>
           <td><%= taskRoles[taskRole].taskStatuses[i].containerIp %></td>
           <td><%= convertGpu(taskRoles[taskRole].taskStatuses[i].containerGpus) %></td>
-          <td><a href="<%= taskRoles[taskRole].taskStatuses[i].containerLog %>" target="_blank">Task Log</a></td>
+          <td>
+            <a onclick='showSshInfo("<%= taskRoles[taskRole].taskStatuses[i].containerId %>")'
+<%
+              if (jobStatus.state !== 'RUNNING') {
+%>
```

Comment:
indentation, pls refer to some [ejs examples](http://www.embeddedjs.com/getting_started.html#create_template)

Reply:
Thanks. Will do.